test/e2e/suite_log_test.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2024 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"path/filepath"
+
+	"github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
+	"github.com/karmada-io/karmada/pkg/util/names"
+	"github.com/karmada-io/karmada/pkg/util/restmapper"
+	"github.com/karmada-io/karmada/test/e2e/framework"
+)
+
+const logDir = "/tmp/karmada/objects"
+
+func printAllBindingAndRelatedObjects() {
+	filePath := filepath.Join(logDir, testNamespace+".txt")
+	err := os.MkdirAll(logDir, 0700)
+	gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+	logFile, err := os.OpenFile(filePath, os.O_CREATE|os.O_APPEND|os.O_RDWR, 0600)
+	gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+	defer logFile.Close()
+
```

Comment:
Some cases also need to query the information about the cluter object. Can the information be output synchronously?

Reply:
done